mysite/chat/models.py

```diff
@@ -178,15 +178,25 @@ def get_next_url(self):
         ) if self.chat and self.chat.next_point else None
 
     def get_errors(self):
+        errors = None
         error_list = UnitError.objects.get(id=self.content_id).get_errors()
-        checked_errors = UnitError.objects.get(id=self.content_id).response.studenterror_set.all()\
-                                                                  .values_list('errorModel', flat=True)
-        error_str = '<li><div class="chat-check chat-selectable %s" data-selectable-attribute="errorModel" ' \
-                    'data-selectable-value="%d"></div><h3>%s</h3></li>'
-        errors = reduce(lambda x, y: x+y, map(lambda x: error_str %
-                                              ('chat-selectable-selected' if x.id in checked_errors else '',
-                                               x.id, x.lesson.title), error_list))
-        return '<ul class="chat-select-list">'+errors+'</ul>'
+        if error_list:
+            checked_errors = UnitError.objects.get(
+                id=self.content_id
+            ).response.studenterror_set.all().values_list('errorModel', flat=True)
+            error_str = '<li><div class="chat-check chat-selectable %s" data-selectable-attribute="errorModel" ' \
```

Comment:
* use unicode strings
* don't use back slash for long lines. Use this construction instead:

```
error_str = (
    '<li><div class="chat-check chat-selectable %s" data-selectable-attribute="errorModel" ' 
    
    'data-selectable-value="%d"></div><h3>%s</h3></li>'
) 
```